lib/index.js

```diff
@@ -44,6 +44,10 @@ function startPlayback ({muted, timeout, inline}, elementCallback) {
     } else {
       sendOutput(true)
     }
+    
+    // Clean up to avoid MediaElementLeak
+    element.remove();
+    element.srcObject = null;
```

Comment:
nitpick: looks like this project is currently set up with `standard`, so, trailing semicolons aren't allowed.
```suggestion
    element.remove()
    element.srcObject = null
```